test/java/seedu/addressbook/commands/SortCommandTest.java

```diff
@@ -0,0 +1,58 @@
+package seedu.addressbook.commands;
+
+import org.junit.Test;
+import seedu.addressbook.data.AddressBook;
+import seedu.addressbook.data.person.Person;
+import seedu.addressbook.data.person.UniquePersonList;
+import seedu.addressbook.util.TestUtil;
+
+import java.util.Collections;
+
+import static org.junit.Assert.assertEquals;
+
+public class SortCommandTest {
+
+    private AddressBook addressBook;
+    private UniquePersonList expectedList;
+    private SortCommand sortCommand;
+
+    @Test
+    public void execute(){
```

Comment:
Need a better name for the test. Please check the coding standards on how to name tests. (Hint: 3 part format)